webplugin/js/app/zendesk-chat-service.js

```diff
@@ -358,7 +359,21 @@ function ZendeskChatService() {
             }
             console.log("Resolved conversation on Kommunicate Dashboard", result);
         });
-    }; 
+    };
+
+    _this.updateVisitorPath = () => {
+        try{
+            var currPage = {
+                title: window.parent.document.title,
+                url: window.parent.location.href
+            };
+            zChat.sendVisitorPath(currPage, function(err) { 
+                console.log("An error occurred while updating visitor path ", err)
```

Comment:
do we need to show any notification to the user?

Reply:
no we do not need to show any notification as this is end user facing not customer facing